frontend/libs/studio-components/src/components/StudioExpression/SimplifiedEditor/LogicalExpressionEditor/SubExpression/SubExpressionValueSelector/SubexpressionValueSelector.test.tsx

```diff
@@ -190,6 +190,14 @@ describe('SubexpressionValueSelector', () => {
       await user.click(document.body);
       screen.getByText(texts.errorMessages[ExpressionErrorKey.InvalidComponentId]);
     });
+
+    it('Displays initial error and handles non-existing component ID', () => {
+      const id = 'non-existing-id';
+      renderSubexpressionValueSelector({ value: { ...componentValue, id }, isInEditMode: true });
+      screen.getByText(texts.errorMessages[ExpressionErrorKey.ComponentIDNoLongerExists]);
```

Comment:
We should add an assertion to the get method. 😊 
```suggestion
  expect(screen.getByText(texts.errorMessages[ExpressionErrorKey.ComponentIDNoLongerExists])).toBeInTheDocument();
```